docs: add documentation for adding vocabularies using thesaurus (#131)
wetransform/hc-documentation: docs/setup-hc/edit-metadata-config/2015-02-10-theme-edit-metadata.md

                       "value": "{\"id\":\"geoNutz/20130319\",\"name\":\"Nutzungsbestimmungen für die Bereitstellung von Geo-daten des Bundes\",\"url\":\"https://sg.geodatenzent-rum.de/web_public/gdz/lizenz/geonutzv.pdf\",\"quelle\":\"Quelle: © GeoBasis-DE / BKG (Jahr des letzten Datenbezugs)\"}"
                   }

### Adding thesaurus keywords

> Adding thesaurus keywords to the metadata can be done by defining an `autofillRule` or `defaultValue` in the theme's metadata configuration. The autofill rule can be added as a JSON string with special characters escaped. After running the autofill workflow, it might be necessary to refresh the web page to see the string displayed in the respective fields in the UI. The same approach can be used for `defaultValue`.

Reviewer: ```suggestion
Adding thesaurus keywords to the metadata can be done by defining an `autofillRule` or `defaultValue` in the theme's metadata configuration. The autofill rule can be added as a JSON string with special characters escaped. After running the autofill workflow for a related dataset, it might be necessary to refresh the web page to see the string displayed in the respective fields in the UI.
```

                       "value": "{\"id\":\"geoNutz/20130319\",\"name\":\"Nutzungsbestimmungen für die Bereitstellung von Geo-daten des Bundes\",\"url\":\"https://sg.geodatenzent-rum.de/web_public/gdz/lizenz/geonutzv.pdf\",\"quelle\":\"Quelle: © GeoBasis-DE / BKG (Jahr des letzten Datenbezugs)\"}"
                   }

> ### Adding thesaurus keywords via autofill rule

Reviewer: ```suggestion
### Adding thesaurus keywords
```

> Adding thesaurus keywords to the metadata can be done by defining an autofill rule in the json representation of the metadata config.

Reviewer: ```suggestion
Adding thesaurus keywords to the metadata can be done by defining an autofill or default rule in the theme's metadata configuration.
```

The rule can be added as a json string with special characters escaped. 

> After running the autofill workflow, it might be necessary to refresh the  web page to see the string displayed in the respective fields in the UI.

Reviewer: ```suggestion
After running the autofill workflow, it might be necessary to refresh the web page to see the string displayed in the respective fields in the UI.
```
